File: Programmers/Lv.0/등수 매기기/120882.java
class Solution {
    public int[] solution(int[][] score) {
        int[] rank = new int[score.length];
        
        double[] avg = new double[score.length];
        for (int a=0; a<avg.length; a++) {
            avg[a] = (double) (score[a][0] + score[a][1]) / 2;
        }
        
        for (int i=0; i<avg.length; i++) {
            int r = 1;
            for (int j=0; j<avg.length; j++) {
                if (avg[i] < avg[j]) r++;
            }
            rank[i] = r;
        }
        
        return rank;
    }
}

File: Programmers/Lv.0/등수 매기기/120882.js
function solution(score) {
    const answer = [];
    
    const sum = [];
    score.forEach(s => {
       sum.push(s[0] + s[1]); 
    });
    
    sum.forEach(v1 => {
       answer.push(sum.filter(v2 => v1 < v2).length + 1);
    });
    
    return answer;
}
